sites/intellij-guide/contents/tutorials/getting-started-spring-data-jpa/creating-employee-entity/index.md

```diff
@@ -0,0 +1,100 @@
+---
+type: TutorialStep
+date: 2021-06-02
+title: Creating an Employee Entity
+technologies: []
+topics: []
+author: da
+subtitle: Create a JPA entity representing an Employee.
+thumbnail: ./thumbnail.png
+
+---
+
+An Entity is a Java class representing data you want to insert into a database. For our application, we will create an Employee entity that we will use to insert Employee data into an Employee table in our database.
```

Comment:
Is there a link we can add to "entity" which is like the "real" definition of a JPA entity?

Reply:
We could either add a link to the JPA spec chapter on entities: https://download.oracle.com/otn-pub/jcp/persistence-2_2-mrel-spec/JavaPersistence.pdf?AuthParam=1623026821_dca716f193492ec792f545aa7cdec618#G4.1410611

Or we could link to this page: https://docs.oracle.com/cd/E16439_01/doc.1013/e13981/undejbs003.htm

What do you think?